docs/source/_notebooks/xgboost-titanic.rst

```diff
@@ -0,0 +1,2468 @@
+        <td style="padding: 0 1em 0 0.5em; text-align: right; border: none;">
+            -0.052
+        </td>
+        <td style="padding: 0 0.5em 0 0.5em; text-align: left; border: none;">
+            Pclass=1
+        </td>
+        
+            <td style="padding: 0 0.5em 0 1em; text-align: right; border: none;">
+                Missing
+            </td>
+        
+    </tr>
+            
+                <tr style="background-color: hsl(0, 100.00%, 94.81%); border: none;">
+        <td style="padding: 0 1em 0 0.5em; text-align: right; border: none;">
+            -0.063
+        </td>
+        <td style="padding: 0 0.5em 0 0.5em; text-align: left; border: none;">
+            &lt;BIAS&gt;
+        </td>
+        
+            <td style="padding: 0 0.5em 0 1em; text-align: right; border: none;">
+                1.000
+            </td>
+        
+    </tr>
+            
+                <tr style="background-color: hsl(0, 100.00%, 94.11%); border: none;">
+        <td style="padding: 0 1em 0 0.5em; text-align: right; border: none;">
+            -0.075
+        </td>
+        <td style="padding: 0 0.5em 0 0.5em; text-align: left; border: none;">
+            Age
+        </td>
+        
+            <td style="padding: 0 0.5em 0 1em; text-align: right; border: none;">
+                19.000
+            </td>
+        
+    </tr>
+            
+                <tr style="background-color: hsl(0, 100.00%, 83.30%); border: none;">
+        <td style="padding: 0 1em 0 0.5em; text-align: right; border: none;">
+            -0.333
+        </td>
+        <td style="padding: 0 0.5em 0 0.5em; text-align: left; border: none;">
+            Pclass=3
+        </td>
+        
+            <td style="padding: 0 0.5em 0 1em; text-align: right; border: none;">
+                1.000
+            </td>
+        
+    </tr>
+            
+    
+            </tbody>
+        </table>
+    
+                
+            
+    
+            
+    
+    
+    
+        
+    
+        
+    
+        
+    
+        
+    
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+        
+    
+    
+    
+
+
+
+
+Weight means how much each feature contributed to the final prediction
+across all trees. The idea for weight calculation is described in
+http://blog.datadive.net/interpreting-random-forests/; eli5 provides an
+independent implementation of this algorithm for XGBoost and most
+scikit-learn tree ensembles.
+
+Here we see that classifier thinks it's good to be a female, but bad to
+travel third class. Some features have "Missing" as value (we are
+passing ``show_feature_values=True`` to view the values): that means
+that the feature was missing, so in this case it's good to not have
+embarked in Southampton.
+
+It's possible to show contributions of only features that are present
```

Comment:
Yeah, it sounds strange, I agree! I changed it to "It's possible to show only features that are present using feature_filter argument:" in d044c15